src/SearchboxIndex.ts

```diff
@@ -1,3 +1,9 @@
+if (window['Coveo'] == undefined) {
+  window['Coveo'] = {};
+}
+import Globalize = require('globalize');
+window['Coveo']['Globalize'] = Globalize;
```

Comment:
Humm, I am not a fan of this, but I understand why it needs to be done like that (for our external cultures files, which cannot use a module loader)

I think we should instead use expose-loader, to set Globalize on the global window.